src/app/static/src/app/store/load/mutations.ts

```diff
@@ -1,15 +1,24 @@
 import {Mutation, MutationTree} from 'vuex';
 import {LoadingState, LoadState} from "./load";
 import {PayloadWithType} from "../../types";
-import {Error} from "../../generated";
+import {
+    Error, ProjectRehydrateResultResponse,
+    ProjectRehydrateStatusResponse
+    , ProjectRehydrateSubmitResponse
+} from "../../generated";
 
 type LoadMutation = Mutation<LoadState>
 
 export interface LoadMutations {
     SettingFiles: LoadMutation,
     UpdatingState: LoadMutation,
     LoadFailed: LoadMutation,
-    LoadStateCleared: LoadMutation
+    LoadStateCleared: LoadMutation,
+    PreparingModelOutput: LoadMutation,
+    ModelOutputStatusUpdated: LoadMutation,
+    PollingStatusStarted: LoadMutation,
+    RehydrateResult: LoadMutation,
+    RehydrateResultError: LoadMutation
```

Comment:
These mutations seem a bit inconsistently named - can we stick with 'Rehydrate..` here?
```suggestion
    LoadStateCleared: LoadMutation,
    PreparingRehydrate: LoadMutation,
    RehydrateStatusUpdated: LoadMutation,
    RehydratePollingStarted: LoadMutation,
    RehydrateResult: LoadMutation,
    RehydrateResultError: LoadMutation
```